_sass/components/_nav.scss

```diff
@@ -4,16 +4,40 @@
   }
 }
 
+#sign-in {
```

Comment:
one school of thought I've seen is that using IDs for styles is an anti-pattern because it means things are not repeatable. What do you think of making this like `.sign-in-logo-wrapper` or something as a class? In case we ever want a second button on the page?

Reply:
As I see this element being unique in the long-term, I felt the id attribute was appropriate.  If we ever needed to add another similar element, at that time we'd certainly change the id to a class, but I'm fine doing that now to save an extra step later.